client/my-sites/sidebar/sidebar.jsx

```diff
@@ -345,13 +347,23 @@ export class MySitesSidebar extends Component {
 	};
 
 	store() {
+		const allowedCountryCodes = [ 'US', 'CA' ];
```

Comment:
Hmm, on second thought. Do we really want something in `client/my-sites/sidebar` importing directly from `client/extensions/woocommerce` ??